MVVMExample/MVVMExample/WorkersController.swift

```diff
@@ -27,15 +27,12 @@ class WorkersController: UIViewController {
         searchBar.reactive.continuousTextValues.observeValues {[weak self] (tempText) in
             guard let strongSelf = self else { return }
             guard let text = tempText else { return }
-            
-            strongSelf.viewModel.needfulWorkers = strongSelf.viewModel.workers.flatMap({ (worker) -> Worker? in
-                if worker.name.contains(text) || text == "" {
-                    return worker
+                        
+            strongSelf.viewModel.fetchNeedfulWorkers(text: text, haveChanges: { (isChange) in
```

Comment:
Change fileprivate var viewModel = ViewModel() to fileprivate var viewModel: ViewModelProtocol = ViewModel()